rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/kustomize/mapping/Kustomization.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.rosco.manifests.kustomize.mapping;
+
+import com.fasterxml.jackson.annotation.*;
+import lombok.Data;
+
+import java.nio.file.Path;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Data
+public class Kustomization {
+
+    private List<String> resources = null;
+
+    private List<ConfigMapGenerator> configMapGenerator = null;
+
+    private List<String> cdrs = null;
```

Comment:
Did you mean `crds`? Is crd a kustomize thing?

Reply:
Nope, it should be CRD.

Reply:
replaced